### PR TITLE
Various spelling fixes

### PR DESCRIPTION
**Pull request by comradekingu**

### CODE REVIEW

File: MediaInfoPopover.qml [hunk 1]
```diff
@@ -22,12 +22,12 @@ Popover {
 
             spacing:units.gu(1)
             Label {
-                text:i18n.tr("Media Information");
+                text:i18n.tr("Media info");
```
Comment (comradekingu) [inline MediaInfoPopover.qml]: I translated this as "Program info", but can't seem to remember why…

**Comment by Flohack74** [comment]
Hmm you need to do more in order to fix this:
- After changing ANY localized string, run the build step. It should modify/update the .pot file.
- Commit the updated .pot file as part of this PR

Otherwise Weblate cannot see the cahnges to translations.

**Comment by Flohack74** [comment]
Can you update this PR plz?

**Comment by comradekingu** [comment]
@Flohack74 Done :)

**Comment by Flohack74** [comment]
Hmm now we got a merge conflict :) - You need to resolve that as well, probably by catching up to changes in our master branch.

**Comment by Flohack74** [comment]
Please update to latest master branch changes and resolve the conflict. Thx!

**Comment by Flohack74** [comment]
I close this now due to inactivity...

**Pull request closed without merging**